src/utils/ExtrusionLine.h

```diff
@@ -60,6 +60,11 @@ struct ExtrusionLine
      */
     coord_t getLength() const;
 
+    /*!
+     * Get the minimal width of this path
+     */
+    coord_t getMinimalWidth() const;
```

Comment:
This new function seems to be unused.

Reply:
remnant of an old solution, kept in because it seemed like a logical interface which can be used in the future.

Reply:
I would still remove it in the interest of reducing maintenance burden.

However my opinion about this isn't very strong, because I see your point too. You can decide for yourself if you think it's worth it to keep then.